src/server/error.h

```diff
@@ -39,7 +39,7 @@ using facade::kWrongTypeErr;
   do {                           \
     OpStatus __s = (x).status(); \
     if (__s != OpStatus::OK) {   \
-      return (x).status();       \
```

Comment:
what was the problem with double evaluation here?

Reply:
`RETURN_ON_BAD_STATUS(SomeSlowFunction)`
or 
`RETURN_ON_BAD_STATUS(NonIdempotentFunction)`

Reply:
Oh I see now that RETURN_ON_BAD_STATUS was used sometimes with functions and not with the returned status